mkdocs.yml

```diff
@@ -104,7 +104,8 @@ nav:
       - Extras:
           - extras/intro.md
           - extras/fastapi.md
-          - extras/websocket.md
+          - extras/websocket_client.md
```

Comment:
```suggestion
          - extras/websocket.md
```
The file is named websocket, not websocket_client

Reply:
fixed the name of the file instead